spec/features/visitor/create_spec.rb

```diff
@@ -25,7 +25,7 @@
     fill_in :Zip, with: "66666"
     fill_in :Email, with: "chunky_lover@example.com"
     fill_in :Password, with: "123password"
-    fill_in :user_password_confirmation, with: "123password"
```

Comment:
i think user_password_confirmation is a form_for convention that wasn't working with form_tag

Reply:
good catch